docs/iris/src/whatsnew/contributions_1.11/bugfix_2016-Oct-06_is_circular.txt

```diff
@@ -0,0 +1,2 @@
+- Fixed a bug where the :func:`iris.util._is_circular` would return false where 
```

Comment:
Previously we started each contribution with a `*`, should we keep consistency with that? I know `-` is the correct character, but `*` is also allowed, and mixing them in a single list would be a bad idea so we have to pick one.